docs/CHANGELOG.rst

```diff
@@ -5,15 +5,15 @@ Version 0.5.0
 -------------
 In development.
 
+* Added the 2019 PDG data table, now default.
```

Comment:
Might be worth adding:

```
* Some poorly established particles not in the current PDG data files were previously erroneously made available. They have now been removed.
```

Reply:
Yep. I will add that tomorrow. Thanks for the comments.